src/components/custom/CustomRadioButton.jsx

```diff
@@ -1,34 +1,44 @@
 import React from 'react'
 
-const CustomRadioButton = ({ label_1, label_2, disabled, onChange, value }) => {
+const CustomRadioButton = ({
+  label_1,
+  label_2,
+  dataCy_1,
+  dataCy_2,
+  disabled,
+  onChange,
+  radio_value,
+  value_1,
+  value_2,
```

Comment:
And is there a way for some of these values to be used for the same things, or is that already optimized?

Reply:
I think we can use value and label, but I thought that we may want some flexibility. Maybe premature optimization, but also it is more clear to assign value and label separately.